media-fonts/inconsolata-lgc/inconsolata-lgc-1.1.0_p20140917.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit font
+
+EGIT_COMMIT="d086a618248121d61e3f3de64b4301230d1c860c"
+DESCRIPTION="Modified version of Inconsolata Hellenic adding the Cyrillic alphabet"
+HOMEPAGE="https://github.com/DeLaGuardo/Inconsolata-LGC"
+SRC_URI="https://github.com/DeLaGuardo/Inconsolata-LGC/tarball/${EGIT_COMMIT} -> ${P}.tar.gz"
```

Comment:
Why not simply use the last released tarball, or ask the author for a new release? https://github.com/DeLaGuardo/Inconsolata-LGC/archive/LGC-1.1.0.tar.gz

Reply:
Because, sadly, the last release is behind the latest commit, and it is *after* the last release that the maintainer committed TTF files of the font into the repo.

Reply:
Ok, thanks. Is the author not responsive? I've asked people for new tags before and they promptly did it.

Reply:
I did not attempt to contact the maintainer.

Reply:
@amadio To be honest this is no big deal. We can commit this ebuild as is using a commit hash.